docs/source/intro.rst

```diff
@@ -0,0 +1,122 @@
+************
+Introduction
+************
+
+:Date: 2017-07-06
+:Authors: Chris Fonnesbeck
+:Contact: fonnesbeck@gmail.com
+:Web site: http://github.com/pymc-devs/pymc3
+
+
+Purpose
+=======
+
+PyMC3 is a probabilistic programming module for Python that allows users to fit Bayesian models using a variety of numerical methods, most notably, Markov chain Monte Carlo (MCMC) and variational inference (VI). Its flexibility and extensibility make it applicable to a large suite of problems. Along with core model specification and fitting functionality, PyMC3 includes methods for summarizing output, plotting, goodness-of-fit and model diagnostics.
```

Comment:
The rules for commas in English are a bit of a mystery to me, but do we need the comma after "most notably"?
"suite of problems" sounds a bit strange to me (but there are a lot of google hits, so maybe that is a thing after all). How about something like "It it being used in biology, astronomy, physics, the social sciences, business and finance and can be extended to fit new needs" (we might want to check that list however). I guess that would cover "flexibility".
Maybe "... fitting functionality, PyMC3 has support for visualisation of results and model diagnostics."?